app/controllers/identify_controller.rb

```diff
@@ -121,6 +121,9 @@ def log_certificate(cert)
     }
 
     attributes.delete(:issuer) if validation_result == 'self-signed cert'
+    if valid
+      attributes[:matched_policy_oids] = cert.matched_policy_oids.map { |item| [item, true] }.to_h
+    end
```

Comment:
There's a lint error here with [`Rails/IndexWith`](https://docs.rubocop.org/rubocop-rails/2.7/cops_rails.html#railsindexwith)

```suggestion
    attributes[:matched_policy_oids] = cert.matched_policy_oids.index_with(true) if valid
```

Reply:
I've reversed this commit.

Reply:
Ok, I'm fine with that too. Would the one-liner in the suggestion above work just as well? Dunno if you had a preference.

Reply:
I was talking about this suggestion: https://github.com/18F/identity-pki/pull/437#discussion_r1523290474

```suggestion
    attributes[:matched_policy_oids] = cert.matched_policy_oids.index_with(true) if valid
```

I expect current commit may cause a lint error.

Reply:
I guess the lint was fine.